src/settings/components/MCLPaginationFields/MCLPaginationFields.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { TestForm } from '@folio/stripes-erm-components/test/jest/helpers';
+import userEvent from '@testing-library/user-event';
+import MCLPaginationFields from './MCLPaginationFields';
+
+const onSubmit = jest.fn();
+const mclList = ['agreementLines', 'agreementEresources', 'entitlementOptions', 'packageContents', 'entitlements'];
+
+describe('MCLPaginationFields', () => {
+  test('renders mcl labels', () => {
+    const { getByTestId } = render(
+      <TestForm onSubmit={onSubmit}>
+        <MCLPaginationFields />
+      </TestForm>
+
+    );
+
+    mclList.forEach((mcl) => {
+      expect(getByTestId(mcl)).toBeInTheDocument();
+    });
+  });
+
+  test('type value and render', () => {
```

Comment:
We sould rather have a test that verifies the values on a form submit rather than in the actual TextField. So, you would have to spy on the `onSubmit` and verify if the field returns the expected value. Makes sense?

Reply:
How do we spy in react testing library? I didn't find an example yet.

Reply:
`jest.fn()` can be used to mock the callback.
So you could do 
`onSubmit = jest.fn()` and in your tests, you could enter a text in the TextField and check if `onSubmit` has been called with the expected values using the `toHaveBeenCalledWith` helper.

https://jestjs.io/docs/en/mock-functions#custom-matchers